src/module.d.ts

```diff
@@ -0,0 +1,328 @@
+import * as React from 'react';
+
+interface PropertyBag<T> {
+    [propName: string]: T;
+}
+
+export namespace components {
+
+export interface RowDefinitionProps {
+    //The column value that should be used as the key for the row
+    //if this is not set it will make one up (not efficient)
+    rowKey?: string;
+
+    // TODO: Unused?
+    //The column that will be known used to track child data
+    //By default this will be "children"
+    childColumnName?: string;
+
+    // TODO: Unused?
```

Comment:
A lot of the props I pulled out of `propTypes` seem to be unused. I'm inclined to delete them, at least from the TS definition, if they don't do anything.